### ISSUE TITLE
Create: unit testing

### ISSUE THREAD

**Issue by ChristinaSchmidt1** [saezlab/MetaProViz#32]
This is important and should be good practice!

This is done for each function:
- use default parameters of the function and create input data as part of the unit test by coding. Test different input settings to enable the test of each of the errors, warnings and stops
- use specific parameters individually and also test with different inputs to allow error/warnings for this particular part of the package.
- So basically test every possible line of code of the package

Document that you have done this: create a codecov.yml file, which returns a certificate of how much of your code was tested by unit testing (X%)
